### ISSUE TITLE
Custom prompt message when using unsupported browser

### ISSUE THREAD

**Issue by jakekwak** [Vanessa219/vditor#628]
It would be nice to see some text on the screen when using an unsupported browser.

**Comment by 88250** [comment]
Do you mean to give the user a prompt 'unsupported browser' when using an old browser?

**Comment by jakekwak** [comment]
@88250 yes,  custom message,  it means we can put the message like "unsupported browser, please try to [Google chrome](https://www.google.com/chrome/)".  when click the link,  it will go the download page.

**Comment by Vanessa219** [comment]
Thanks for the suggestion. Browser support requirements and fallback messages should be handled by the host application, since they vary between integrations. An unsupported browser may be unable to parse or execute the Vditor bundle, so Vditor itself cannot reliably display this prompt. The host application can use feature detection and fallback HTML before loading Vditor, and provide its own localized upgrade instructions. Therefore, we do not plan to implement this in Vditor core.
